### DISCUSSION TITLE
AutoExport

### DISCUSSION THREAD

**Discussion by dm5n** [retorquere/zotero-better-bibtex#2121] [Q&A] [answered]
Hi, thanks for the great plugin.

Re: autoexport, I'm exporting the library in Better CSL JSON and Better BibLaTeX formats for use with Obsidian and Zettlr. I often add new references to Zotero as I'm writing, but find that I must manually export using the 'Export Now' button in Better BibTex prefs, to cite just added references.

I've tried with Automatic Export set to both 'On Change' and 'When Idle'. Is there a better way to speed up the exporting of new references in Better Bibtex?

Alternatively, would it be possible to add an Export Now button to the Zotero toolbar itself? 

Thanks.

**Comment by retorquere** [comment, marked as answer]
auto-export is supposed to do just that, so if it isn't working for you, you will have to open an issue so I can look into it.
